backend/app/main/service/family_service.py

```diff
@@ -64,4 +64,5 @@ def get(relative, distance, year, citations):
     family = []
     for key in dictionary:
         family.append(dictionary[key])
+    family = sorted(family, key=lambda paper: paper['year'])
```

Comment:
hmm, might be faster if we do it inside the query. Should be noted for future reference if we get into performance issues